src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -241,6 +241,8 @@
   "minecolonies.config.fishingroddurabilityadjustt2.comment": "Offset for the maximum durability unenchanted rod a T2 Fishing Hut can use, compared to iron tools (250). Thermal Foundation Iron requires +6. T3 huts can use unenchanted rods of any durability level, so long as they can take damage at all. Anything below -250 will only allow vanilla rods, except as allowed by T1.",
   "minecolonies.config.enabledmodtags": "Enabled Mod Tags",
   "minecolonies.config.enabledmodtags.comment": "List of recognized and enabled tags.",
+  "minecolonies.config.auditcraftingtags": "Audit Crafting Tags",
+  "minecolonies.config.auditcraftingtags.comment": "Generate crafting_audit.csv on loading recipes to verify tags.",
```

Comment:
bit weak of a desc for the users, maybe "If enabled crafting dumps crafting item and citizen crafting recipes into crafting_audit.csv on loading"

Reply:
I don't think that's any clearer than what I already wrote.

Reply:
well then come up with something else better :)